types/cookie-parser/cookie-parser-tests.ts

```diff
@@ -4,6 +4,10 @@ import cookieParser = require('cookie-parser');
 const app = express();
 app.use(cookieParser('optional secret string'));
 
+app.use((req: express.Request) => {
+    const secret: string | undefined = req.secret;
+});
```

Comment:
This is better tested like this:
```suggestion
const req = null as any as express.Request;
req.secret; // $ExpectType string | undefined
```